systems/registry/network/pkg/db/net_repo.go

```diff
@@ -21,14 +21,12 @@ import (
 type NetRepo interface {
 	Add(network *Network, nestedFunc func(*Network, *gorm.DB) error) error
 	Get(id uuid.UUID) (*Network, error)
-	GetByName(orgName string, network string) (*Network, error)
-	GetByOrg(orgID uuid.UUID) ([]Network, error)
+	GetByName(network string) (*Network, error)
 	GetAll() ([]Network, error)
-	GetDistinctOrg() ([]uuid.UUID, error)
 	// GetByOrgName(orgName string) ([]Network, error)
 	// Update(orgId uint, network *Network) error
-	Delete(orgName string, network string) error
-	GetNetworkCount(orgID uuid.UUID) (int64, error)
+	Delete(network string) error
```

Comment:
Since we no longer use orgName, then I think we can delete by netId   `delete(id string)`